Header  header
float32[] position
float32[] orientation
float32[] speed
bool[]  reset    
bool    active
bool    calc_orient
bool    gripper